plugins/catalog-backend-module-gcp/src/providers/GkeEntityProvider.ts

```diff
@@ -142,8 +154,14 @@ export class GkeEntityProvider implements EntityProvider {
             [ANNOTATION_KUBERNETES_API_SERVER_CA]:
               cluster.masterAuth?.clusterCaCertificate || '',
             [ANNOTATION_KUBERNETES_AUTH_PROVIDER]: 'google',
-            'backstage.io/managed-by-location': location,
-            'backstage.io/managed-by-origin-location': location,
+            [ANNOTATION_KUBERNETES_DASHBOARD_APP]: 'gke',
+            [ANNOTATION_LOCATION]: location,
+            [ANNOTATION_ORIGIN_LOCATION]: location,
+            [ANNOTATION_KUBERNETES_DASHBOARD_PARAMETERS]: JSON.stringify({
+              projectId: project,
+              region: cluster.location,
+              clusterName: cluster.name,
```

Comment:
I think we should type this if possible, it should be possible with a [typescript Union](https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html), for example:

```
{
  type: "gke",
  parameters: {
     projectId: string,
     region: string,
     clusterName: string,
  }
}

```

Then we could create another `type` for other providers at a later data

Reply:
Great idea. Should I follow up with changes to later layer and change dashboard parameters type in cluster details or only in catalog layer?
This would make the change much bigger. I also might need some more help :)

Reply:
I think we could focus on the catalog and translate them in the cluster details for now and add some TODOs to do the full change in a follow up

Reply:
Been thinking about it.
Should `ANNOTATION_KUBERNETES_DASHBOARD_APP` be deprecated then?
as we have type in dashboard params.
How would I go about it?
Delete it completely, or add deprecation?
@mclarke47